frontend/src/theme/ApiDemoPanel/Curl/index.js

```diff
@@ -0,0 +1,300 @@
+import React, { useRef, useState, useEffect } from 'react'
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext'
+import clsx from 'clsx'
+import codegen from 'postman-code-generators'
+import Highlight, { defaultProps } from 'prism-react-renderer'
+import { useTypedSelector } from '@theme/ApiDemoPanel/hooks'
+import buildPostmanRequest from '@theme/ApiDemoPanel/buildPostmanRequest'
+import FloatingButton from '@theme/ApiDemoPanel/FloatingButton'
+import styles from 'docusaurus-theme-openapi/lib/theme/ApiDemoPanel/Curl/styles.module.css'
+
+const languageSet = [
+  {
+    tabName: 'cURL',
+    highlight: 'bash',
+    language: 'curl',
+    variant: 'curl',
+    options: {
+      longFormat: false,
+      followRedirect: true,
+      trimRequestBody: true,
+    },
+  },
+  {
+    tabName: 'Node',
+    highlight: 'javascript',
+    language: 'nodejs',
+    variant: 'axios',
+    options: {
+      ES6_enabled: true,
+      followRedirect: true,
+      trimRequestBody: true,
+    },
+  },
+  {
+    tabName: 'Go',
+    highlight: 'go',
+    language: 'go',
+    variant: 'native',
+    options: {
+      followRedirect: true,
+      trimRequestBody: true,
+    },
+  },
+  {
+    tabName: 'Python',
+    highlight: 'python',
+    language: 'python',
+    variant: 'requests',
+    options: {
+      followRedirect: true,
+      trimRequestBody: true,
+    },
+  },
+]
+const languageTheme = {
+  plain: {
+    color: 'var(--ifm-code-color)',
```

Comment:
I'm guessing we've pulled/copied this from somewhere, and no concerns with that, but did want to note that the specifics of the args being passed to these function calls (I presume) are a complete black box to me :shrug:

Reply:
Yep. So this is one of the components from `docusaurus-theme-openapi` I have customised.

Its basically a copy of
https://github.com/cloud-annotations/docusaurus-openapi/blob/0beb11b248e34e2b88f171406fc4fb04a390f065/packages/docusaurus-theme-openapi/src/theme/ApiDemoPanel/Curl/index.tsx with some customisations.

I think this is mostly covered in
https://github.com/badges/shields/pull/9014#discussion_r1142363731